<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="2" ignored="0" total="5" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="132273" started-at="2018-09-29T06:19:33Z" finished-at="2018-09-29T06:21:45Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="132273" started-at="2018-09-29T06:19:33Z" finished-at="2018-09-29T06:21:45Z">
      <class name="LandingPage.LpageRepo">
        <test-method status="PASS" signature="LandingPgBrowserOpen()[pri:0, instance:LandingPage.LpageRepo@4b9e13df]" name="LandingPgBrowserOpen" duration-ms="119593" started-at="2018-09-29T06:19:33Z" finished-at="2018-09-29T06:21:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LandingPgBrowserOpen -->
        <test-method status="PASS" signature="LandingPgAcronymclick()[pri:0, instance:LandingPage.LpageRepo@4b9e13df]" name="LandingPgAcronymclick" duration-ms="11779" started-at="2018-09-29T06:21:33Z" depends-on-methods="LandingPage.LpageRepo.LandingPgBrowserOpen" finished-at="2018-09-29T06:21:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LandingPgAcronymclick -->
        <test-method status="FAIL" signature="LandingPgComp()[pri:0, instance:LandingPage.LpageRepo@4b9e13df]" name="LandingPgComp" duration-ms="377" started-at="2018-09-29T06:21:44Z" depends-on-methods="LandingPage.LpageRepo.LandingPgAcronymclick" finished-at="2018-09-29T06:21:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Synchronous Data Link Control] but found [Google Search]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Synchronous Data Link Control] but found [Google Search]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at LandingPage.LpageRepo.LandingPgComp(LpageRepo.java:112)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LandingPgComp -->
        <test-method status="FAIL" signature="LandingPgInvent()[pri:0, instance:LandingPage.LpageRepo@4b9e13df]" name="LandingPgInvent" duration-ms="438" started-at="2018-09-29T06:21:45Z" depends-on-methods="LandingPage.LpageRepo.LandingPgAcronymclick" finished-at="2018-09-29T06:21:45Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed
at org.testng.Assert.fail(Assert.java:96)
at LandingPage.LpageRepo.LandingPgInvent(LpageRepo.java:120)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LandingPgInvent -->
        <test-method status="SKIP" signature="LandingPgInventdummy()[pri:0, instance:LandingPage.LpageRepo@4b9e13df]" name="LandingPgInventdummy" duration-ms="0" started-at="2018-09-29T06:21:45Z" depends-on-methods="LandingPage.LpageRepo.LandingPgComp" finished-at="2018-09-29T06:21:45Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LpageRepo.LandingPgInventdummy()[pri:0, instance:LandingPage.LpageRepo@4b9e13df] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LpageRepo.LandingPgInventdummy()[pri:0, instance:LandingPage.LpageRepo@4b9e13df] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:887)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LandingPgInventdummy -->
      </class> <!-- LandingPage.LpageRepo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
